tools/cli/commands/install.js

```diff
@@ -104,3 +104,15 @@ export function installDefine( yargs ) {
 
 	return yargs;
 }
+
+/**
+ * Conditionally add the root install task
```

Comment:
Except there's no condition that actually does so here. You still have the conditional logic in the caller.

Reply:
hmm what I was thinking there. Must have been called away.

Reply:
[`995a47b` (#18504)](https://github.com/Automattic/jetpack/pull/18504/commits/995a47b2fd1b59ff686fc514e2af902f84c0c38c)